test/unit/uk/gov/hmrc/apinotificationpull/util/XmlBuilderSpec.scala

```diff
@@ -16,16 +16,25 @@
 
 package uk.gov.hmrc.apinotificationpull.util
 
+import org.mockito.Mockito._
+import org.scalatest.mockito.MockitoSugar
+import uk.gov.hmrc.apinotificationpull.config.AppContext
 import uk.gov.hmrc.apinotificationpull.model.Notifications
-import uk.gov.hmrc.apinotificationpull.util.XmlBuilder.toXml
 import uk.gov.hmrc.play.test.UnitSpec
 
-class XmlBuilderSpec extends UnitSpec {
+class XmlBuilderSpec extends UnitSpec with MockitoSugar {
+
+  trait Setup {
+    val apiContext = "notifications"
+    val appContext = mock[AppContext]
+    when(appContext.apiContext).thenReturn(apiContext)
+    val xmlBuilder = new XmlBuilder(appContext)
+  }
 
   "XmlBuilder.toXml()" should {
 
```

Comment:
I am confused too. Commit message says
```
The queue service returns /notification/{notificationId} as a notification location. We need to expose this to the client as /notifications/{notificationId} so the path is correct for this service, notifications being the context of this service
```
But code is changing plural `notifications` to `notification`

Reply:
This is the test, the change to from `notifications` to `notification` is to emulate what the queue returns to us. The logic change is then in the `XmlBuilder` which now constructs the path using the `apiContext` from config and the `notificationId` extracted from the path returned by the queue.